### DISCUSSION TITLE
컴포넌트란 무엇인가요?

### DISCUSSION THREAD

**Discussion by greenblues1190** [woowacourse-study/2022-woowahan-bansanghwe#24] [React]
https://reactjs.org/docs/components-and-props.html

**Comment by airman5573** [comment]
노스님 말에 의하면, 실제로 면접때 이런 질문이 나왔다고 합니다.

**Comment by InKyoJeong** [comment]
오,, 갑자기 컴포넌트란 뭐냐고 하면 당황스러울것 같네여

**Comment by airman5573** [comment]
컴포넌트란 **View의 Spec 문서**라는 생각이 듭니다.  
어떤 상황에서 어떻게 그려져야 하는지에 대한 명세인 것이지요.
